tao_compiler/mlir/disc/transforms/fusion_utils.h

```diff
@@ -202,9 +205,13 @@ class FusionPattern {
   SmallVector<Operation*, 4>& getRootOps() { return root_ops_; }
```

Comment:
From both the comments and codes I get confused with the difference between "root_ops_" and "results_". Is this some legacy issue? @wyzero